### DISCUSSION TITLE
May I use DateRangePicker without time ?

### DISCUSSION THREAD

**Discussion by Nizhurin** [rsuite/rsuite#3432] [General]
Hello everyone!

May I use DateRangePicker without time and how ?  or if not possible, how to make the time not adjusted to utc? or maybe is components daterangepicker only for date? Thank you for answer

**Comment by SevenOutman** [comment]
Do you mean that you want DateRangePicker to emit Date object that represents 00:00 of the date you choose, in UTC timezone?
